_posts/2019-02-02-digitalocean-one-click.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Announcing DigitalOcean's OpenFaaS One-click"
+description: Earlier this week saw the release of the first OpenFaaS One-click application on DigitalOcean.  Richard Gee guides you through just how simple it is to create your first OpenFaaS function on DigitalOcean.
+date: 2019-02-02
+image: /images/digitalocean-one-click/balance-macro-ocean.jpg
+categories:
+  - developer-experience
+  - swarm
+  - python
+author_staff_member: richard
+dark_background: false
+---
+
+I've written previously about how you can be [up and running in 5 minutes](/blog/deploy-digitalocean-ansible.md) with OpenFaaS on DigitalOcean.  With the news this week that DigitalOcean are now offering an OpenFaaS One-click application this time shortens further and increases the number of cost effective ways in which to run OpenFaaS on DigitalOcean:
```

Comment:
Explain that we collaborated with DigitalOcean to provide the one-click image. (this makes it more deliberate and shows it was a partnership)